src/scripts/main.js

```diff
@@ -1 +1,72 @@
 'use strict';
+
+const body = document.querySelector('body');
+const div = document.createElement('div');
+let leftClickHappened = false;
+let rightClickHappened = false;
+
+div.setAttribute('data-qa', 'notification');
+body.appendChild(div);
+
+const showNotification = (message, type) => {
+  const notification = document.createElement('div');
+
+  notification.className = type;
```

Comment:
Directly setting the 'className' will overwrite any existing classes. If you want to add a class, consider using 'classList.add' to ensure that you do not unintentionally remove existing classes.